src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -246,12 +246,12 @@ trait AttributeComponent {
                 set a.value_string=ta.value_string, a.value_number=ta.value_number, a.value_boolean=ta.value_boolean, a.value_entity_ref=ta.value_entity_ref
```

Comment:
this query and the one above don't need to be changed because they were already limiting a.owner_id to the set of owner ids